app/views/templates/cover_letter_template.scala.html

```diff
@@ -496,40 +490,97 @@ <h2 class="heading-medium margin-top-small no-margin-bottom print-header__headin
+                    @messages("pdf.ruling.section.ifYouDisagree.cannot")
                 </p>
-                <h4 class="heading-small">Asking for a review with HMRC</h4>
+                <h4 class="govuk-heading-s">@messages("pdf.ruling.section.askForReview.title")</h4>
                 <p class="govuk-body-s">
-                    Email <strong>@messages("pdf.ruling.email")</strong> asking for a review and give your reasons within 30 days of receiving your ruling email.
+                    @messages("pdf.ruling.section.askForReview.reviewOfficer")
                 </p>
                 <p class="govuk-body-s">
-                    If you accept an offer for a review of the ruling decision, it will be carried out by an HMRC officer who has not been involved in your case. You will be able to give any evidence and further information and the review officer will email you to tell you the outcome of the review.
+                    @messages("pdf.ruling.section.askForReview.ifYouDisagree")
                 </p>
-                <h4 class="heading-small">Appealing to an independent tribunal</h4>
                 <p class="govuk-body-s">
-                    You must contact the tribunal within 30 days of your ruling email to discuss your case. If you disagree with your review from HMRC, then you must contact the tribunal within 30 days of that outcome.
+                    @messages("pdf.ruling.section.askForReview.furtherInfo")
                 </p>
+                <h4 class="govuk-heading-s">@Html(messages("pdf.ruling.section.askForReview.guidance.url"))</h4>
                 <p class="govuk-body-s">
-                    HMRC is not able to give a 3 month extension for this appeal, it can only be given by the independent tribunal.
+                    @messages("pdf.ruling.section.askForReview.requestReview")
                 </p>
-                <p class="govuk-body-s mt-40">
-                    Yours sincerely,
+                <p class="govuk-body-s">
+                    @messages("pdf.ruling.section.askForReview.solsTeam")
+                </p>
+                <ul class="govuk-list govuk-list--bullet govuk-body-s">
+                    <li>@messages("pdf.ruling.section.askForReview.solsTeam.1")</li>
+                    <li>@messages("pdf.ruling.section.askForReview.solsTeam.2")</li>
+                    <li>@messages("pdf.ruling.section.askForReview.solsTeam.3")</li>
+                    <li>@messages("pdf.ruling.section.askForReview.solsTeam.4")</li>
+                    <li>@messages("pdf.ruling.section.askForReview.solsTeam.5")</li>
+                </ul>
+                <p class="govuk-body-s">
+                    @messages("pdf.ruling.section.askForReview.solsTeam.cannotWrite")
+                </p>
+                <p class="govuk-body-s">
+                    @Html(messages("pdf.ruling.section.askForReview.solsTeam.emailText"))
```

Comment:
This should be `pdf.ruling.section.askForReview.solsTeam.email` not `pdf.ruling.section.askForReview.solsTeam.emailText`. `pdf.ruling.section.askForReview.solsTeam.emailText` has been removed so it is not rendering.